documentation/asciidoc/computers/raspberry-pi/boot-eeprom.adoc

```diff
@@ -13,10 +13,13 @@ If an error occurs during boot, then an xref:configuration.adoc#led-warning-flas
 
 There are multiple ways to update the bootloader of your Raspberry Pi.
 
-==== Raspberry Pi 5, Raspberry Pi 4 and Raspberry Pi 400
+==== Raspberry Pi 5, Compute Module 5, Raspberry Pi 500, Raspberry Pi 4 and Raspberry Pi 400
```

Comment:
Is CM4 intentionally missing?

Reply:
A few lines further down it says "N.B. On Compute Module 4 and CM4S the bootrom is unable to load the `recovery.bin` file from EMMC and automatic bootloader updates are not enabled.
The recommended update mechanism is `rpiboot` or via `flashrom` - see `rpi-eeprom-update -h` for more information."

Should that N.B. be turned into a NOTE: block @nathan-contino ?

Reply:
Fixed!